### DISCUSSION TITLE
Can i have the messenger.com as an app ?

### DISCUSSION THREAD

**Discussion by DareDevil2000** [tw93/Pake#773] [Q&A]
I was looking for an utility that can turn the (messenger.com) as an web based app like the (WhatsApp web app) so I just came across this utility , So does this can do that ?

**Comment by tw93** [comment]
Maybe you can try `pake https://www.messenger.com/ --name Messenger `

**Reply by k1m1nonawa4869** [reply]
Hi the author, I want to ask, how can I put my own logo for Messenger if using Pake via GitHub Actions? Thank you.

**Reply by faishalkc** [reply]
You can read a documentation by just adding --icon "path to ico file"

**Reply by k1m1nonawa4869** [reply]
> You can read a documentation by just adding --icon "path to ico file"

sorry for disturbing you but how can I add icon file .ico from my computer to Github Actions. When I tried that it shows "No files were found with the provided path: node_modules/pake-cli/output/*. No artifacts will be uploaded."
